app/src/main/java/com/example/flyman3046/allnews/DetailActivity.java

```diff
@@ -0,0 +1,97 @@
+package com.example.flyman3046.allnews;
+
+import android.app.ProgressDialog;
+import android.os.AsyncTask;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+import android.util.Log;
+import android.view.MenuItem;
+import android.widget.TextView;
+
+import com.example.flyman3046.allnews.Model.DataConstants;
+
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.select.Elements;
+
+import java.io.IOException;
+
+/**
+ *
+ */
+public class DetailActivity extends AppCompatActivity {
+    private static final String TAG = DetailActivity.class.getSimpleName();
+
+    private ProgressDialog mProgressDialog;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.article_detail);
+
+        Toolbar myToolbar = (Toolbar) findViewById(R.id.article_detail_toolbar);
+        setSupportActionBar(myToolbar);
+        if(getSupportActionBar() != null) {
+            getSupportActionBar().setDisplayShowTitleEnabled(false);
```

Comment:
keep it as optional, since main purpose is to learn instead of creating a maintainable app.